docs/python/modelspec/programmatic/topology/geometry.rst

```diff
@@ -854,11 +859,24 @@ Reading 3D Data from NEURON
         a logical connection point if the translation ruins the 
         visualization. 
          
+        pt3d information is stored as 32 bit floating point numbers. These
+        numbers have about 7 decimal digits of resolution. In contrast, most
+        other values in NEURON are double precision and have about 17 decimal
```

Comment:
Probably doesn't matter, but as you know, doubles can't guarantee 17 digits of precision because 1+1e-16 == 1 even though 1 + 2e-16 > 1. Is it better to say about 17 digits or about 16 digits?

Reply:
Changed to 16